Tests/ArcGISToolkitTests/LocationButtonTests.swift

```diff
@@ -51,12 +56,12 @@ struct LocationButtonTests {
         #expect(button.contextMenuAutoPanOptions == [.off, .recenter])
         
         // Test when `off` is last
-        button = button.autoPanModes([.recenter, .compassNavigation, .off])
```

Comment:
For the testcases below, I simply removed or swapped `.compassNavigation` to let the testcase build on all platforms with small deviation to what the testcase tests originally.

If we want to keep the original testcase, I can add the compiler block instead.